Source/WebCore/dom/ViewTransition.cpp

```diff
@@ -260,6 +262,34 @@ static ExceptionOr<void> checkDuplicateViewTransitionName(const AtomString& name
     return { };
 }
 
+static RefPtr<ImageBuffer> snapshotNodeVisualOverflowClippedToViewport(LocalFrame& frame, Node& node, SnapshotOptions&& options, LayoutRect& oldOverflowRect)
+{
+    if (!node.renderer())
+        return nullptr;
+
+    auto paintBehavior = frame.view()->paintBehavior();
+    auto backgroundColor = frame.view()->baseBackgroundColor();
+
+    options.flags.add(SnapshotFlags::Accelerated);
+
+    frame.view()->setBaseBackgroundColor(Color::transparentBlack);
+    frame.view()->setNodeToDraw(&node);
+
+    ASSERT(node.renderer()->hasLayer());
```

Comment:
This demo hits the assert FYI: https://codepen.io/argyleink/full/abPOmgj

Reply:
Fixed, was missing `RenderTableRow::requiresLayer`.